src/conf_mode/system_console.py

```diff
@@ -129,12 +128,8 @@ def apply(console):
         # Configure screen blank powersaving on VGA console
         call('/usr/bin/setterm -blank 15 -powersave powerdown -powerdown 60 -term linux </dev/tty1 >/dev/tty1 2>&1')
 
-    # Start getty process on configured serial interfaces
-    for device in console['device']:
-        # Only start console if it exists on the running system. If a user
-        # detaches a USB serial console and reboots - it should not fail!
-        if os.path.exists(f'/dev/{device}'):
-            call(f'systemctl restart serial-getty@{device}.service')
+    # Service control moved to op_mode/restart_serial.py to unify checks and prompts. 
+    call('/usr/libexec/vyos/op_mode/restart_serial.py --quiet')
```

Comment:
Please de-hardcode the path and refer to https://github.com/vyos/vyos-1x/blob/current/python/vyos/defaults.py#L24